NitroxLauncher/LauncherLogic.cs

```diff
@@ -358,6 +358,16 @@ private void SyncAssembliesBetweenSubnauticaManagedAndLib()
 
             List<string> ignoreNoBinaries = new List<string>();
             CopyAllAssemblies(libDirectory, subnauticaManagedPath, ignoreNoBinaries);
+
+            CopyLogConfig(libDirectory);
+        }
+
+        private void CopyLogConfig(string libDirectory)
+        {
+            string logConfig = "NLog.config";
```

Comment:
If possible it would be better to use NLog API to set the configuration of the logger(s). Relying on the launcher to copy this file over is less certain.